### ISSUE TITLE
write logfile

### ISSUE THREAD

**Issue by 150d** [bwssytems/ha-bridge#1120]
Hello,

is it possible to have HA-Bridge write a logfile? This would make troubleshooting easier than to follow console output.

Regards

**Comment by bwssytems** [comment]
On windows and linux you can have the output redirected to a file:

[cmd] > test.txt 2>&1

If you use services on linux to start the ha-bridge it automatically goes into the syslog.

**Comment by 150d** [comment]
Yes, I'm aware, but my experience with redirection (using Windows) isn't great. You always get some program that doesn't use stdout but writes directly, you don't get to read prompts anymore either, you name it.

It's just not the most reliable solution.
